contracts/governance/Staking/Staking.sol

```diff
@@ -373,6 +477,46 @@ contract Staking is
         emit StakingWithdrawn(msg.sender, amount, until, receiver, isGovernance);
     }
 
+    /**
+     * @notice Send user' staked tokens to a receiver.
+     * This function is dedicated only for governance vesting withdrawal.
+     *
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
+     * @param receiver The receiver of the tokens. If not specified, send to the msg.sender
+     * @param vesting The vesting address which act as the caller here.
```

Comment:
the vesting is not a caller here and unlocked tokens are not processed
pls go through comments and natspec and fix irrelevant